sunkit_instruments/goes_xrs/goes_xrs.py

```diff
@@ -1394,7 +181,19 @@ def flux_to_flareclass(goesflux: u.watt / u.m**2):
     return f"{str_class}{goes_subclass:.3g}"
 
 
-def _assert_chrono_order(obstime):
-    chrono_check = obstime[1:] - obstime[:-1]
-    if not all(val > TimeDelta(0 * u.day) for val in chrono_check):
-        raise ValueError("Elements of obstime must be in chronological order.")
+def calculate_temperature_em(*args, **kwargs):
+    raise NotImplementedError(
+        "This functionality is now provided through sunkit_instruments.goes_xrs.calculate_temperature_em"
+    )
+
+
+def calculate_radiative_loss_rate(*args, **kwargs):
+    raise NotImplementedError(
+        "The functionality to calculate the radiative loss has not yet been implemented in sunkit_instruments"
+    )
+
+
+def calculate_xray_luminosity(*args, **kwargs):
+    raise NotImplementedError(
+        "The functionality to calculate the X-ray luminosity has not yet been implemented in sunkit_instruments"
+    )
```

Comment:
I think I would remove these and we just add a changelog entry saying what was removed and what was added (with function names).